packages/slate-editor/src/modules/editor-v4-floating-add-menu/lib/shouldShowMenuButton.ts

```diff
@@ -17,6 +17,9 @@ export function shouldShowMenuButton(editor: Editor): boolean {
     }
 
     const text = Node.string(currentNode);
+    const hasPlaceholder = currentNode.children.some(
+        (child) => 'type' in child && child.type === PLACEHOLDER_NODE_TYPE,
```

Comment:
Could you use Array.from(Node.nodes(currentNode)) and isPlaceholderNode

Reply:
I went with @e1himself solution